miso-web/src/main/webapp/scripts/list_samples.js

```diff
@@ -16,6 +16,8 @@ ListTarget.sample = (function () {
       } else if (config.requisitionId) {
         if (config.supplemental) {
           return Urls.rest.samples.requisitionSupplementalDatatable(config.requisitionId);
+        } else if (config.filter === "samples-prepared") {
+          return Urls.rest.samples.samplesPreparedDatatable(config.requisitionId);
         } else {
           return Urls.rest.samples.requisitionDatatable(config.requisitionId);
         }
```

Comment:
This is now using 2 parameters to control 1 thing. Switch to work the same as the library table - `config.relation` with value "requisitioned", "supplemental", or "indirect"